RaspberryPi/WebApp/scripts/home.php

```diff
@@ -0,0 +1,22 @@
+<!-- 
+    Title: HOME command.
+    Date: 8/1/2016
+    Author: Robert Reinert
+    Description: Sends the Rover's current mission to go home.
+
+-->
+
+
+<?php
+
+$url = "http://localhost:9000/home";
```

Comment:
Can you use a variable (based upon a configuration file entry)instead of localhost (which defaults to localhost)? That would make it easier to move the web app to another device/platform (such as an Android app).